### PR TITLE
feature: add set mac address for pouch cli

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
add set mac address for pouch cli.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
added

### Ⅳ. Describe how to verify it
ci pass

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

**Comment by ZYecho** [comment]
LGTM.

**Comment by allencloud** [comment]
I found that MacAddress is already defined in swagger.yml. While I am afraid that we miss the MAC address validation rules there.

I found that mac address is already supported in go-swagger https://github.com/go-swagger/go-swagger/blob/master/generator/formats.go#L50

And you can update the MacAddress in https://github.com/alibaba/pouch/blob/master/apis/swagger.yml#L2190 with a regex rule which refer to https://stackoverflow.com/questions/4260467/what-is-a-regular-expression-for-a-mac-address.

@rudyfly @ZYecho

**Comment by allencloud** [comment]
We can also use IPv6 in https://github.com/go-swagger/go-swagger/blob/master/docs/use/spec/strfmt.md to validate IPv6 field in swagger. FYI @rudyfly

File: test/cli_run_network_test.go [hunk 1]
```diff
@@ -74,3 +75,26 @@ func checkPortMapExists(c *check.C, portMap types.PortMap, port string) {
...
+
+	command.PouchRun("run", "-d", "--name", cname, "--mac-address", macAddress, busyboxImage, "sleep", "1000").Assert(c, icmd.Success)
+	defer command.PouchRun("rm", "-vf", cname)
+
+	res := command.PouchRun("exec", cname, "ip", "addr", "show")
+	res.Assert(c, icmd.Success)
+
+	stdout := res.Stdout()
+	found := false
+	for _, line := range strings.Split(stdout, "\n") {
```
Comment (fuweid) [inline test/cli_run_network_test.go]: why not just use contains here?

**Comment by rudyfly** [comment]
@allencloud next PR to do it.

**Pull request merged**